modules/tinymce/src/core/test/ts/browser/html/StylesTest.ts

```diff
@@ -235,35 +242,12 @@ describe('browser.tinymce.core.html.StylesTest', () => {
     assertStyles(styles, 'background:url(vbscript:alert(1)', `background: url('vbscript:alert(1');`);
   });
 
-  it('TINY-9819: force_hex_color set to "off" (the default)', () => {
+  it('TINY-10436: HEX colours are enforced always', () => {
     const styles = Styles();
-    for (const color of [
-      '#aabbcc',
-      'rgb(1, 2, 3)',
-      'rgba(1, 2, 3, 0.5)',
-      `rgba(200, 150, 100, 0.95)`
-    ]) {
-      // All colors are unchanged:
-      assertStyles(styles, `color: ${color};`, `color: ${color};`);
-    }
-  });
-
-  it('TINY-9819: force_hex_color set to "always"', () => {
-    const styles = Styles({ force_hex_color: 'always' });
     assertStyles(styles, 'color: #aabbcc;', 'color: #aabbcc;');
     assertStyles(styles, 'color: rgb(1, 2, 3);', 'color: #010203;');
     assertStyles(styles, 'color: rgba(1, 2, 3, 1);', 'color: #010203;');
     assertStyles(styles, 'color: rgba(1, 2, 3, 0);', 'color: #010203;');
     assertStyles(styles, 'color: rgba(1, 2, 3, 0.5);', 'color: #010203;');
```

Comment:
So we are just throwing away the alpha channel? Is this really correct. Should it be hex rgba or skipped if it has a alpha? So loading up a document now with an alpha channel on colors will change them all to new colors when you save so data is lost seems a bit radical. This is how it worked in v5: https://fiddle.tiny.cloud/8RqRc8d4X1/0

The rgba is ignored the rgb is converted. With this PR these two colors become the same color.

- [ ] Consider skipping alpha channel or at least do RGBA hex

Reply:
So how about this (similar to v5)?
```ts
// Convert RGB colors to HEX
RgbaColour.fromString(value)
  .filter((rgba) => rgba.alpha === 1)
  .each((rgba) => {
    value = Transformations.rgbaToHexString(RgbaColour.toString(rgba)).toLowerCase();
  });
```
```ts
const styles = Styles();
assertStyles(styles, 'color: #aabbcc;', 'color: #aabbcc;');
assertStyles(styles, 'color: #aabbcc88', 'color: #aabbcc88;');
assertStyles(styles, 'color: #aabbccff', 'color: #aabbccff;');
assertStyles(styles, 'color: rgb(1, 2, 3);', 'color: #010203;');
assertStyles(styles, 'color: rgba(1, 2, 3, 0);', 'color: rgba(1, 2, 3, 0);');
assertStyles(styles, 'color: rgba(1, 2, 3, 0.5);', 'color: rgba(1, 2, 3, 0.5);');
assertStyles(styles, 'color: rgba(1, 2, 3, 1);', 'color: #010203;');
// Exactly what v5 would do:
// assertStyles(styles, 'color: rgba(1, 2, 3, 1);', 'color: rgba(1, 2, 3, 1);');
```

Or do RGBA Hex, which I guess would need an update to the relevant function in Acid?

Reply:
Make it behave like v5. That was my intent with this change all along!

Reply:
It should now behave exactly like it did in v5